packages/compartment-mapper/test/fixtures-conditional-host-exports/node_modules/app/index.js

```diff
@@ -0,0 +1 @@
+export { feature } from 'lib';
```

Comment:
why is `app` under `node_modules`?

Reply:
Ah, this would mostly work at the directory containing `node_modules`, but the compartment mapper issues a warning if the `package.json` `name` does not match the parent directory name.